bee-network/bee-gossip/src/service/event.rs

```diff
@@ -123,8 +131,20 @@ pub enum InternalEvent {
         substream: Box<NegotiatedSubstream>,
     },
 
-    /// The gossip protocol has been dropped with a peer.
-    ProtocolDropped { peer_id: PeerId },
+    /// The gossip protocol with a peer was stopped.
+    ProtocolStopped { peer_id: PeerId },
```

Comment:
`peer_id` is not documented (for consistency)

Reply:
Fixed